webscrambles/src/net/gnehzr/tnoodle/server/webscrambles/OrderedScrambles.java

```diff
@@ -0,0 +1,245 @@
+                            scrambleRequest.roundStartTime = activityStartTime;
+                            scrambleRequestListByDay.get(index).add(scrambleRequest);
+                        }
+                    }
+                }
+
+                for (int index = 0; index<dayList.size(); index++) {
+                    if (scrambleRequestListByDay.get(index).size()>0) {
+                        
+                        // Since we are iterating over the schedule, this did not seem necessary,
+                        // but it was for some cases (like the order of MBLD for WC2019).
+                        Collections.sort(scrambleRequestListByDay.get(index));
+                        
+                        String pdfFileName = "Printing/Ordered Scrambles/";
+
+                        if (hasMultipleVenues) {
+                            pdfFileName += venueName.replace('/', ' ')+"/";
+                        }
+
+                        if (hasMultipleDays || dayList.size() > 1) {
+                            pdfFileName += "Day "+dayList.get(index)+"/";
+                        }
+
+                        pdfFileName += "Ordered Scrambles";
+                        
+                        // In addition to different folders, we stamp venue, day and room in the PDF's name
+                        // to prevent different files with the same name.
+                        if (hasMultipleVenues) {
+                            pdfFileName += " - " + venueName.replace('/', ' ');
+                        }
+                        if (hasMultipleDays || dayList.size() > 1) {
+                            pdfFileName += " - Day "+dayList.get(index);
+                        }
+                        
+                        if (hasMultipleRooms) {
+                            pdfFileName += " - "+roomName.replace('/', ' ');
+                        }
+                        pdfFileName += ".pdf";
+                        
+                        // removing slashes may add double+ extra spaces
+                        pdfFileName = pdfFileName.replaceAll("\\s+", " ");
+
+                        parameters.setFileNameInZip(pdfFileName);
+                        zipOut.putNextEntry(null, parameters);
+                        ScrambleRequest[] scrambleRequests = scrambleRequestListByDay.get(index).toArray(new ScrambleRequest[scrambleRequestListByDay.get(index).size()]);
+                        ByteArrayOutputStream baos = ScrambleRequest.requestsToPdf(globalTitle, generationDate, scrambleRequests, null);
+                        zipOut.write(baos.toByteArray());
+                        zipOut.closeEntry();
+                    }
+                }
+            }
+        }
+    }
+
+    private static boolean compareFirstCharToNumber(String letter, int number) {
+        return letter.charAt(0)-'A' == number-1;
+    }
+
+    private static String removeQuotation(String s) {
```

Comment:
This feels weird to me. Why are quotation marks showing up in these strings?

Reply:
Fixed. No removeQuotation is used anymore.